include/eve/concept/properly_convertible.hpp

```diff
@@ -19,29 +20,30 @@ namespace eve
   {
     // Note: this traits is customizable for other types we may need later on (complex,etc...)
     template<typename T, typename U>
-    struct  has_no_loss_convertion
-          : std::bool_constant<!(std::is_floating_point_v<T> && std::is_integral_v<U>)>
-    {};
+    struct has_no_loss_convertion
+        : std::bool_constant<!(std::is_floating_point_v<T> && std::is_integral_v<U>)>
+    {
+    };
```

Comment:
Can we not do the 2 line braces? There should be an option.

Reply:
Fixed